cosmwasm-plugin/main.go

```diff
@@ -0,0 +1,296 @@
+package main
+
+import (
+	"embed"
+	"encoding/gob"
+	"fmt"
+	"io"
+	"io/fs"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	hplugin "github.com/hashicorp/go-plugin"
+
+	"github.com/ignite/cli/ignite/services/chain"
+	"github.com/ignite/cli/ignite/services/plugin"
+)
+
+var (
+	chainName  string
+	chainNamed string
+	//go:embed wasm-wiring/*
+	templates embed.FS // Embedded template files
+
+)
+
+func init() {
+	gob.Register(plugin.Manifest{})
+	gob.Register(plugin.ExecutedCommand{})
+	gob.Register(plugin.ExecutedHook{})
+}
+
+type p struct{}
+
+func (p) Manifest() (plugin.Manifest, error) {
+	return plugin.Manifest{
+		Name: "cosmwasm-plugin",
+		// Add commands here
+		Commands: []plugin.Command{
+			// Example of a command
+			{
+				Use:   "cosmwasm-plugin",
+				Short: "Explain what the command is doing...",
+				Long:  "Long description goes here...",
+				Flags: []plugin.Flag{
+					{Name: "my-flag", Type: plugin.FlagTypeString, Usage: "my flag description"},
+				},
```

Comment:
```suggestion
```

Reply:
Removed long description and flags: https://github.com/ignite/plugins/commit/79f099d58090d75447f38aab06a8c5701d1bd64f